Wiki-Become-A-Mod.md

```diff
@@ -1,6 +1,6 @@
 # Requirements to become a wiki Moderator
 
-There are many ways to contribute to the [wiki](Wiki) without being part of the [Senior Group](Senior-Wiki-Mods). However, if you want to become one, there are certain requirements that you must meet first:
+There are many ways to contribute to the [wiki](Wiki) without being part of the [Senior Group](Wiki-Senior-Mods). However, if you want to become one, there are certain requirements that you must meet first:
```

Comment:
It will link to a list of the wiki mods, I just haven't created it yet. Hoping for a solution for the mod's group dilema first, otherwise it will jsut point to the current Mod's team.

The idea was to have the `Seniors` with merge rights and `Juniors` without them but still able to assign people and handle labels and issues.